src/core/Akka.MultiNodeTestRunner.Shared/Sinks/TimelineLogCollectorActor.cs

```diff
@@ -14,25 +14,28 @@
 using Akka.Actor;
 using Akka.Event;
 using Akka.MultiNodeTestRunner.Shared.Reporting;
-using Akka.Util.Internal;
 
 namespace Akka.MultiNodeTestRunner.Shared.Sinks
 {
     public class TimelineLogCollectorActor : ReceiveActor
     {
         private readonly SortedList<DateTime, HashSet<LogMessageInfo>> _timeline = new SortedList<DateTime, HashSet<LogMessageInfo>>();
-        
-        public TimelineLogCollectorActor()
+        private readonly LogLevel _minimumLogLevel;
+
+        public TimelineLogCollectorActor(string minimumLogLevel)
         {
+            LogMessageInfo.TryParseLogLevel(minimumLogLevel, out _minimumLogLevel);
```

Comment:
Yep, this should do the trick! Nice work!

Now, when I am looking at it, it seems like log level setting is specific to `PrintToConsole` command handling. That is, when receiving `DumpToFile` message we do not apply any filtering based on the log level provided. And that's great by the way - we want to reduce amount of logs printed in the console, but for debugging perposes (and for some MNTR reports) it is nice to have all logs saved without filtering.

So, this might be a good idea to pass required log level as an optional parameter of `PrintToConsole` command - do you agree? That would make it clear that we are just filtering console output.

Reply:
Makes sense - modified accordingly